pages/tools/Pages.md

```diff
@@ -1,22 +1,25 @@
 ---
 title: Federalist
 redirect_from:
-  - /federalist/
+  - /pages/
```

Comment:
I'm going to request changes as well. I _think_ we shouldn't remove any `redirect_from` entries because they're meant to ensure that previous links continue to work. Instead of removing `/federalist/` and adding `/pages/`, I think we should keep `/federalist/` and add `/pages/tools/federalist/`, so it would look like:

```yaml
redirect_from:
  - /federalist/
  - /pages/tools/federalist/
```

Since `/pages/` has never existed before, we don't need to redirect from it.